src/main/app/case/case.ts

```diff
@@ -140,8 +140,8 @@ export interface Case {
   applicant1KnowsApplicant2Address?: YesOrNo;
   iWantToHavePapersServedAnotherWay?: Checkbox;
   legalProceedings?: YesOrNo;
-  legalProceedingsDetails?: string;
-  legalProceedingsRelated?: LegalProceedingsRelated[];
+  legalProceedingsByCase?: ListValue<LegalProceeding>[];
```

Comment:
this might be a problem, as LegalProceeding is an object which also has the properties CaseNumber/CaseDetails.